### PR TITLE
DEVXP-2787 - Log silent return when pypi info is not found

### PR DESCRIPTION
**Pull request by vschoener**

### CODE REVIEW

File: libyear/libyear [hunk 1]
```diff
@@ -3,7 +3,7 @@ import argparse
 
 from prettytable import PrettyTable
 
-from libyear.pypi import get_lib_days, get_no_of_releases
+from libyear.pypi import get_lib_days
```
Comment (vschoener) [inline libyear/libyear:7]: Not used

File: libyear/pypi.py [hunk 2]
```diff
@@ -65,12 +68,13 @@ def get_version_release_dates(name, version, version_lt):
...
     version = get_version(pypi_data, version)
     if version is None:
+        logging.warning(f'No Pypi version "{version}" for package "{name}"')
         return None, None, None, None
 
     try:
         latest_version_date = releases[latest_version][-1]['upload_time_iso_8601']
     except IndexError:
-        print(f'Latest version of {name!r} has no upload time.')
+        logging.info(f'Latest version of {name!r} has no upload time.')
```
Comment (vitorquintanilha) [inline libyear/pypi.py]: Does this cause a package to be skipped? If it does, then maybe we should use `warning`  to call our attention that a pkg is being skipped

Comment (vitorquintanilha) [reply libyear/pypi.py]: Ah nevermind, it see it doesn't!

Comment (vschoener) [reply libyear/pypi.py]: I hesitated with a warning for all except. I can do the switch if it makes more sense.

**Pull request merged**